src/components/UI/Notes/Notes.tsx

```diff
@@ -0,0 +1,140 @@
+  const [noteDataSet, SetNoteDataSet] = useNotes();
+  console.log(data);
+  var initialStateNote = {
+    title: props.title,
+    htmlbody: props.content,
+    createdOn: "",
+    color: props.color,
+    priority: props.priority,
+    label: props.label,
+    id: props._id,
+    pin : props.pin
+  };
+  function EditNoteHandler() { 
+    data.noteReducer({ type: "reset", data: initialStateNote });
+    data.showNoteToggle(true);
+    
+  }
+
+  async function PinNoteHandler() { 
+    // data.noteReducer({ type: "reset", data: {
+    //   title: props.title,
+    //   htmlbody: props.content,
+    //   createdOn: "",
+    //   color: props.color,
+    //   priority: props.priority,
+    //   label: props.label,
+    //   id: props._id,
+    //   pin : props.pin === VAR_NotPinnedNotes ? VAR_PinnedNotes : VAR_NotPinnedNotes
+    // } });
+    // data.showNoteToggle(true);
+    var object = {
+			title: props.title,
+			content: props.content,
+			color: props.color,
+			createdOn: props.cr,
+			priority: props.priority,
+			pin: props.pin === VAR_NotPinnedNotes ? VAR_PinnedNotes : VAR_NotPinnedNotes,
+			label: props.label,
+    };
+    
+		console.log(object,props._id);
+		var res = await axiosRequest({
+			method: "post",
+			url: "/api/notes/"+props._id,
+			data: { note: object },
+			headers: {
+				authorization: localStorage.getItem(VAR_ENCODE_TOKEN),
+			},
+		});
+		console.log(res);
+    SetNoteDataSet(res.notes);
+  }
+
+  function ArchiveHandler() { 
+    try {
+			(async () => {
+				var res = await  axiosRequest({
+          method: "post",
+          url: "/api/notes/archives/" +  props._id,
+          headers: {
+            authorization: localStorage.getItem(VAR_ENCODE_TOKEN)
+          },
+          data: { note: { ...props } }
+        
+        });
+        console.log(res,res.archives,setArchiveContextArray);
+        setArchiveContextArray(res.archives);
+        SetNoteDataSet(res.notes);
+			})();;
+		} catch (error) {
+			console.log("Product list page error", error);
+			// Alert("error", "Some error occured!! refresh page and try again");
+		}
+  }
+
+  function TrashHandler() { 
+    try {
+			(async () => {
+				var res = await  axiosRequest({
+          method: "delete",
+          url: "/api/notes/" +  props._id,
+          headers: {
+            authorization: localStorage.getItem(VAR_ENCODE_TOKEN)
+          }
+        
+        });
+        console.log(res);
+        setTrashContextArray((prev)=>[...prev,props]);
+			})();;
+		} catch (error) {
+			console.log("Product list page error", error);
+			// Alert("error", "Some error occured!! refresh page and try again");
+		}
+  }
+  return (
+    <div className='note-details-container' style={{backgroundColor : props.color  || "wheat"}}>
+      <h3>{props.title}</h3>
+      <div className='note-pin'>
+        {props.pin === VAR_NotPinnedNotes ?
+          <span onClick={()=>PinNoteHandler()}><BiPinAngle /> </span>
```

Comment:
Variable naming should not be mixed. Instead of ```VAR_NotPinnedNotes```, you can use ```PINNED_NOTES```

Reply:
I kept the prefix as var for knowing it isa global const